ee/packages/pdf-worker/src/templates/ChatTranscript/components/Message.tsx

```diff
@@ -23,20 +23,58 @@ const styles = StyleSheet.create({
 	},
 });
 
-const messageLongerThanPage = (message: string | undefined) => (message?.length ?? 0) > 1200;
+const MAX_MD_ELEMENTS_PER_VIEW = 10;
+const MAX_MSG_SIZE = 1200;
+
+const messageLongerThanPage = (message: string | undefined) => (message?.length ?? 0) > MAX_MSG_SIZE;
+
+// When a markup list is greater than 10 (magic number, but a reasonable small/big number) we're gonna split the markdown into multiple <View> element
+// So react-pdf can split them evenly across pages
+const markupEntriesGreaterThan10 = (messageMd: any[]) => messageMd.length > MAX_MD_ELEMENTS_PER_VIEW;
+const splitByTens = (array: any[] = []): any[][] => {
+	const result = [];
+	for (let i = 0; i < array.length; i += 10) {
```

Comment:
Maybe we can use the constant you defined here? instead of the hardcoded `10`?

Also, could we replace `any` for `unknown`?